charts/stable/pod-gateway/values.yaml

```diff
@@ -135,11 +144,13 @@ probes:
     enabled: false
 
 service:
-  type: ClusterIP
-  clusterIP: None
-  port:
-    port: 4789
-    protocol: UDP
+  main:
+    ports:
+      http:
+        type: ClusterIP
+        clusterIP: None
+        port: 4789
+        protocol: UDP
 
 webhook:
```

Comment:
Please add some helm-docs to this section (and relevant sub-sections) as well. We are trying to make it so that (most) of the keys in the values.yaml has some info about what it does.